TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -394,7 +433,11 @@ void TTSSpeaker::createPipeline() {
     }
 
     // set the TTS volume to max.
+#if defined(PLATFORM_REALTEK)
+    g_object_set(G_OBJECT(m_audioVolume), "volume", (double) (m_defaultConfig.volume() / MAX_VOLUME), NULL);
```

Comment:
could you confirm if volume element expects the "volume" value between 0-1?

Reply:
realtek also expects volume between rang 0-1
My custom build was verified by @mark.yang